h/static/scripts/widget-controller.js

```diff
@@ -38,6 +39,27 @@ module.exports = function WidgetController(
 ) {
 
   /**
+   * Returns the number of top level annotations which are of type annotations
+   * and not notes or replies.
+   */
+  function countAnnotations(annotations) {
+    var totalAnnotations = annotations.reduce(function(annotationsCount, annotation) {
```

Comment:
It is usually easier to read a function if local variable names are short. It would be fine to use `total` and `count` rather than `totalAnnotations` and `annotationsCount` here.